src/mono/mono/mini/aot-compiler.c

```diff
@@ -6111,6 +6114,90 @@ method_is_externally_callable (MonoAotCompile *acfg, MonoMethod *method)
 	}
 }
 
+#ifdef MONO_ARCH_AOT_SUPPORTED
+static gboolean
+method_to_pinvoke_has_scope_import (MonoAotCompile *acfg, MonoMethod *method, const char **module, const char **entrypoint)
```

Comment:
nit: I guess we could just keep the old name for this function, its static (private to this source file) and all callsites has been updated.